api/src/main/resources/workbench.yaml

```diff
@@ -2040,7 +2074,50 @@ definitions:
         description: >
           All the selected values in the data set
         items:
-          $ref: "#/definitions/ValueSet"
+          $ref: "#/definitions/DomainValuePair"
+
+  DomainValuePair:
+    type: object
+    properties:
+      domain:
+        description: >
+          Domain corresponding to an OMOP table that can contain rows for the concepts in this
```

Comment:
It was pointed out to me that most of this description we don't need, since its just copy paste. (Maybe `Domain corresponding to an OMOP table`?